### PR TITLE
feat: change download section to be installer section

### PR DESCRIPTION
**Pull request by ErickSharp**
Changes the functionality of the "Download" section to be solely dedicated for the installer. Also adds new spacings to the tailwind config.

Before:
image

After:
image

### CODE REVIEW

File: src/components/home/Installer.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import InstallerImage from '../../assets/img/InstallerScreenshot.png';
+
+export const Installer: React.FC = () => {
+    return (
+        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-gray-100 ">
+            <div className="ml-20  mt-10 max-w-2xl flex flex-col items-center xl:items-start text-center xl:text-left -mb-20 xl:mb-20 space-y-4">
+                <div className="inline-flex items-center justify-center p-3 bg-blue-light-contrast rounded-md shadow-lg">
+                    <FontAwesomeIcon icon={faBoxOpen} size="2x" />
+                </div>
+                <h1 className="text-5xl pt-4 text-teal-light-contrast font-bold">Custom-built Installer</h1>
```
Comment (armankir) [inline src/components/home/Installer.tsx:15]:
```suggestion
                <h1 className="text-5xl pt-4 text-gray-700 font-bold">Custom-built Installer</h1>
```

**Review by armankir — CHANGES_REQUESTED** [review]
Would fit better with the other white section if the title was black.

File: src/components/home/Installer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                <h1 className="text-xl text-black">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis delectus facilis et, totam architecto nisi.</h1>
```
Comment (armankir) [inline src/components/home/Installer.tsx:16]:
```suggestion
                <h1 className="text-xl text-black">The FlyByWire Installer makes installing, managing, and updating FlyByWire projects easier then ever, and it's all at the click of one button.</h1>
```

**Review by armankir — CHANGES_REQUESTED** [review]
Please add text, what I added is all I could think of, you can change as you see fit.

File: src/components/home/Installer.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '../Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBoxOpen } from '@fortawesome/free-solid-svg-icons';
+// TODO: Update image when installer redesign is finished.
+import InstallerImage from '../../assets/img/InstallerScreenshot.png';
+
+export const Installer: React.FC = () => {
+    return (
+        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-gray-100 ">
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:10]:
```suggestion
        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-gray-100">
```
Remove space

File: src/components/home/Installer.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            <div className="ml-20  mt-10 max-w-2xl flex flex-col items-center xl:items-start text-center xl:text-left -mb-20 xl:mb-20 space-y-4">
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:11]:
```suggestion
            <div className="ml-20 mt-10 max-w-2xl flex flex-col items-center xl:items-start text-center xl:text-left -mb-20 xl:mb-20 space-y-4">
```
Remove space

File: src/components/home/Installer.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+                <div className="inline-flex items-center justify-center p-3 bg-blue-light-contrast rounded-md shadow-lg">
+                    <FontAwesomeIcon icon={faBoxOpen} size="2x" />
+                </div>
+                <h1 className="text-5xl pt-4 text-teal-light-contrast font-bold">Custom-built Installer</h1>
+                <h1 className="text-xl text-black">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis delectus facilis et, totam architecto nisi.</h1>
+                <a href="https://api.flybywiresim.com/installer">
+                    <Button className="w-40 mt-4 bg-teal-light-contrast border-2 border-teal-light-contrast hover:bg-white hover:text-teal-light-contrast  font-bold">Download</Button>
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:18]:
```suggestion
                    <Button className="w-40 mt-4 bg-teal-light-contrast border-2 border-teal-light-contrast hover:bg-white hover:text-teal-light-contrast font-bold">Download</Button>
```
Remove space

File: src/components/home/Installer.tsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+                </a>
+            </div>
+            <div className="relative w-5/6 xl:-right-144 -bottom-40 shadow-xl">
+                <img src={InstallerImage} alt="Installer Image" />
+            </div>
+
+        </div>
+    );
+};
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:27]: Entire component does not need to have return function as there's nothing logical inside of it. Simply change {} to () and remove return.

Comment (ErickSharp) [reply src/components/home/Installer.tsx:27]: Good call. Didn't know that.

**Comment by ErickSharp** [comment]
Updated it to be dark-themed for Arman. We are just waiting on the installer redesign to be finished and new shadows to be added by Holland since they look like shit at 2xl which is what we need for them to be even noticeable atm.

File: src/components/home/Installer.tsx (hunk 1 above)
Comment (ErickSharp) [reply src/components/home/Installer.tsx]: Made the changes to make it dark

File: src/components/home/Installer.tsx (hunk 6 above)
Comment (NathanInnes) [reply src/components/home/Installer.tsx]: If you could change it to what I mentioned that would be great

**Comment by NathanInnes** [comment]
Refrain from using offensive/swears in commit messages. Please amend.

**Review by armankir — DISMISSED** [review]

File: src/components/home/Installer.tsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Button } from '../Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBoxOpen } from '@fortawesome/free-solid-svg-icons';
+// TODO: Update image when installer redesign is finished.
+import InstallerImage from '../../assets/img/InstallerScreenshot.png';
+
+export const Installer: React.FC = () => {
+    return (
+        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark ">
```
Comment (NathanInnes) [inline src/components/home/Installer.tsx:10]:
```suggestion
        <div className="relative h-144 w-screen flex flex-col xl:flex-row items-center overflow-hidden bg-blue-dark">
```
Remove additional space.

**Comment by ErickSharp** [comment]
Moving this to another PR due to conflicts.

**Pull request closed without merging**